docs/development/codeguide.rst

```diff
@@ -135,8 +135,17 @@ Coding Style/Conventions
   <https://www.python.org/dev/peps/pep-0008/>`_. In particular, this includes
   using only 4 spaces for indentation, and never tabs.
 
+  * ``astropy`` itself enforces this style guide using the
```

Comment:
Maybe we should mention the PR or hash when this was "activated" in the branch? Or maybe that doesn't matter?

Didn't @eteq suggest some special tag?